setup/setup-roles.sql

```diff
@@ -76,3 +76,12 @@ CREATE ROLE cwa_federation_download
    NOCREATEROLE
    NOREPLICATION
    IN ROLE cwa_user;
+  
```

Comment:
I assume that this user is only for the sgs table. I think it would also make sense to have AC here. So that this swiss Role cannot have access to the other efgs table. What is your opinion on this?